### PR TITLE
students export to csv

### PR DESCRIPTION
**Pull request by Mudassar045**
Side work feature for MIS

### CODE REVIEW

File: web/src/modules/Exports/style.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	justify-content: center;
+}
+
+.exports-modal .row .button {
+	padding: 10px 15px;
+	margin-top: 10px;
+	margin-left: auto;
+}
+
+@media (max-width: 768px) {
```
Comment (taimur38) [inline web/src/modules/Exports/style.css:21]: I like the designing for 3 screen widths instead of 2

File: web/src/modules/Student/List/index.tsx [hunk 2]
```diff
@@ -13,6 +13,8 @@ import { StudenPrintableIDCardList } from 'components/Printable/Student/cardlist
 import getSectionsFromClasses from 'utils/getSectionsFromClasses'
 import { StudentIcon } from 'assets/icons'
 import { chunkify } from 'utils/chunkify'
+import Modal from 'components/Modal'
+import StudentExportModal from 'modules/Exports/studenExportModal'
```
Comment (taimur38) [inline web/src/modules/Student/List/index.tsx:17]: Should we put this in settings next to CSV import instead?

File: web/src/utils/downloadCSV.ts [hunk 3]
```diff
@@ -11,4 +13,15 @@ const downloadCSV = (data: string[][], name: string) => {
 	hiddenElem.click();
 }
 
-export default downloadCSV;
\ No newline at end of file
+export const downloadAsCSV = (data: any, headers: string[], filename = "csv-generated-by-mischool") => {
```
Comment (taimur38) [inline web/src/utils/downloadCSV.ts:16]: What's the difference between the 2 methods?

**Comment by Mudassar045** [comment]
> Nice work - you made this on your own as a side task?

Yes

File: web/src/modules/Student/List/index.tsx (hunk 2 above)
Comment (Mudassar045) [reply web/src/modules/Student/List/index.tsx:17]: Yes, exports should be on a same page.

File: web/src/utils/downloadCSV.ts (hunk 3 above)
Comment (Mudassar045) [reply web/src/utils/downloadCSV.ts:16]: Second method is using `papaparse` utility to generate csv using `data` and `headers` separately.

**Pull request merged**